packages/xstate-react/src/useMachine.ts

```diff
@@ -9,8 +9,7 @@ import {
 import { useActor } from './useActor.ts';
 
 /**
- *
- * @deprecated Use `useActor(...)` instead.
+ * @alias useActor
```

Comment:
what kind of an effect on the overall in-IDE experience does it have?

Reply:
Right now, it looks like `useMachine` is invalid because it is crossed out (~~`useMachine`~~) which can cause a lot of confusion because it's by far the most popular hook in `@xstate/react`.

If we treat it as an alias, we prevent confusion.